artcommon/artcommonlib/konflux/konflux_db.py

```diff
@@ -216,7 +216,7 @@ async def get_latest_build(
             completed_before: typing.Optional[datetime] = None,
             extra_patterns: dict = {},
             strict: bool = False,
-    ) -> typing.Optional[KonfluxRecord]:
+    ) -> typing.Optional[KonfluxBuildRecord]:
```

Comment:
I think this was supposed to be `KonfluxBuildRecord` since we are retrieving the build from the table, hence made the change, so that I can access the embargo status here: https://github.com/openshift-eng/art-tools/pull/1339/files#diff-bc471e2aee7ba5ac9bc27eca1f0ddc68963a7045561de4be6a3c8a77e0362364R102

Reply:
This method is also used to query `KonfluxBundleBuildRecord`. That is the reason why we use the parent class `KonfluxRecord` here.